jena-arq/src/main/java/org/apache/jena/sparql/vocabulary/FOAF.java

```diff
@@ -19,242 +19,282 @@
-     *  work for.</p>
-     */
-    public static final Property workplaceHomepage = m_model.createProperty( "http://xmlns.com/foaf/0.1/workplaceHomepage" );
-    
-    /** <p>Title (Mr, Mrs, Ms, Dr. etc)</p> */
-    public static final Property title = m_model.createProperty( "http://xmlns.com/foaf/0.1/title" );
-    
-    /** <p>A weblog of some thing (whether person, group, company etc.).</p> */
-    public static final Property weblog = m_model.createProperty( "http://xmlns.com/foaf/0.1/weblog" );
-    
-    /** <p>A logo representing some thing.</p> */
-    public static final Property logo = m_model.createProperty( "http://xmlns.com/foaf/0.1/logo" );
-    
-    /** <p>An organization funding a project or person.</p> */
-    public static final Property fundedBy = m_model.createProperty( "http://xmlns.com/foaf/0.1/fundedBy" );
-    
+    public static final Property based_near = M_MODEL.createProperty( "http://xmlns.com/foaf/0.1/based_near" );
+
+    /** <p>The birthday of this Agent, represented in mm-dd string form, eg. '12-31'.</p> */
+    public static final Property birthday = M_MODEL.createProperty( "http://xmlns.com/foaf/0.1/birthday" );
+
+    /** <p>A current project this person works on.</p> */
+    public static final Property currentProject = M_MODEL.createProperty( "http://xmlns.com/foaf/0.1/currentProject" );
+
     /** <p>A depiction of some thing.</p> */
-    public static final Property depiction = m_model.createProperty( "http://xmlns.com/foaf/0.1/depiction" );
-    
-    /** <p>Indicates a homepage of the service provide for this online account.</p> */
-    public static final Property accountServiceHomepage = m_model.createProperty( "http://xmlns.com/foaf/0.1/accountServiceHomepage" );
-    
-    /** <p>A page or document about this thing.</p> */
-    public static final Property page = m_model.createProperty( "http://xmlns.com/foaf/0.1/page" );
-    
-    /** <p>A document that this thing is the primary topic of.</p> */
-    public static final Property isPrimaryTopicOf = m_model.createProperty( "http://xmlns.com/foaf/0.1/isPrimaryTopicOf" );
-    
-    /** <p>The surname of some person.</p> */
-    public static final Property surname = m_model.createProperty( "http://xmlns.com/foaf/0.1/surname" );
-    
+    public static final Property depiction = M_MODEL.createProperty( "http://xmlns.com/foaf/0.1/depiction" );
+
+    /** <p>A thing depicted in this representation.</p> */
+    public static final Property depicts = M_MODEL.createProperty( "http://xmlns.com/foaf/0.1/depicts" );
+
+    /** <p>A checksum for the DNA of some thing. Joke.</p> */
+    @Deprecated
+    public static final Property dnaChecksum = M_MODEL.createProperty( "http://xmlns.com/foaf/0.1/dnaChecksum" );
+
+    /** <p>The family name of some person.</p> */
+    public static final Property familyName = M_MODEL.createProperty( "http://xmlns.com/foaf/0.1/familyName" );
+
+    /** <p>The family name of some person.</p> */
+    @Deprecated
```

Comment:
For deprecated terms like this where there is a new alternative can we explicitly point to the new term to help users migrate forwards

Reply:
@rvesse that's a good idea. I'll make it so.